app/components/GameScore/index.js

```diff
@@ -47,62 +107,37 @@ class GameScore extends React.PureComponent { // eslint-disable-line react/prefe
 
   checkSwitchServer() {
     const { score1, score2 } = this.state;
-
     if ((score1 + score2) % 2 === 1) {
       this.props.changeServer();
     }
   }
 
-  addPoint(event, isPlayerLeft) {
+  updatePoint(isPlayerLeft, removePoint) {
     if (this.state.wonGame) {
       return;
     }
-
     this.checkSwitchServer();
-
+    const point = removePoint ? -1 : 1;
     const isPlayer1 = isPlayerLeft ? 'score1' : 'score2';
     // SetState is asynchronous, so we have callback.
-    this.setState({ [isPlayer1]: this.state[isPlayer1] + 1 }, () => this.checkGameFinished());
+    this.setState({ [isPlayer1]: this.state[isPlayer1] + point }, () => this.checkGameFinished());
   }
 
   render() {
     return (
       <div>
-        <h2>Game Scores</h2>
-
-        {/* Player 1 */}
-        <div>
-          {this.props.currentGame.player1.name} score: {this.state.score1}
-          <button
-            onClick={(event) => this.addPoint(event, true)}
-            style={{ background: 'grey', marginLeft: 10 }}
-          >
-            point 1+
-          </button>
-          {
-            this.state.wonGame === this.state.player1Name &&
-            <span>Winner</span>
-          }
-        </div>
-
-        {/* Player 2 */}
-        <div>
-          {this.props.currentGame.player2.name} score: {this.state.score2}
-          <button
-            onClick={(event) => this.addPoint(event, false)}
-            style={{ background: 'grey', marginLeft: 10 }}
-          >
-            point 1+
-          </button>
-          {
-            this.state.wonGame === this.state.player2Name &&
-            <span>Winner</span>
-          }
-        </div>
-
+        <GameScoreDisplay>
+          <GameScoreNUmber>{this.state.score1}</GameScoreNUmber>
+          <GameScoreNUmber>{this.state.score2}</GameScoreNUmber>
+        </GameScoreDisplay>
         {
           this.state.wonGame &&
-          <button onClick={() => this.resetGame()} style={{ background: 'green', marginTop: 20 }}>New game</button>
+          <OverLay>
+            <div>
+              <h3>{this.props.currentGame[this.state.wonGame].name} won</h3>
+              <p>Press enter to start next game.</p>
+            </div>
+          </OverLay>
```

Comment:
We should look to use a material-ui dialog here. http://www.material-ui.com/#/components/dialog